app/src/components/BannerPromo.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+    <div v-show="product" class="banner-block">
+        <el-row justify="center" :gutter="50">
+            <el-col :xs="24" :sm="24" :md="8" :lg="6">
+                    <div style="display:inline-flex;">
+                        <div class="pico">
+                            <img :src="product.img">
+                        </div>
+                        <div style="margin-left:20px">
+                            <p class="go-further">{{ translations.goFurther }}</p>
+                            <span class="product-name">{{ product.displayName }}</span>
+                            <p class="addons">{{ translations.addonsMarketplace }}</p>
+                        </div>
+                    </div>
+            </el-col>
+            <el-col :xs="24" :sm="24" :md="16" :lg="12">
+                <p>{{ product.fullDescription }}</p>
+                <slot/>
+            </el-col>
+            <el-col :xs="24" :sm="24" :md="24" :lg="6" class="center">
+                <el-button type="text" @click="dialogDiscover = true"><i class="material-icons" style="vertical-align:middle">computer</i> {{ translations.screenshots }}</el-button>
+                <el-button type="primary" @click="redirectToAddons()">{{ translations.discover }}</el-button>
+            </el-col>
+        </el-row>
+
+        <el-dialog id="dialog-preview-addons" :visible.sync="dialogDiscover">
+            <el-carousel id="screenshot-preview-addons" :interval="5000" height="800px" arrow="always" indicator-position="none">
+                <el-carousel-item v-for="picture in product.pictures" :key="parseInt(picture)">
+                    <img :src="picture.big" width="" height="">
+                </el-carousel-item>
+            </el-carousel>
+            <el-row class="dialog-preview-addons-footer" type="flex" justify="center">
+                <el-col :xs="12" :sm="12" :md="12" :lg="12">
+                    <el-button class="by-prestashop" type="text" @click="dialogDiscover = true"><i class="material-icons" style="vertical-align:middle">check_circle</i> {{ translations.developedBy }}</el-button>
+                </el-col>
+                <el-col :xs="12" :sm="12" :md="12" :lg="12">
+                    <el-button class="right" type="primary" @click="redirectToAddons()">{{ translations.discoverOn }}</el-button>
+                </el-col>
+            </el-row>
+        </el-dialog>
+    </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+const translations = JSON.parse(bannerPromoTranslations)
+
+export default {
+    name: 'BannerPromo',
+    data: function () {
+        return {
+            dialogDiscover: false,
+            product: '',
+            translations: translations
+        }
+    },
+    props: {
+        id_product_addons: Number,
+        iso_code: String,
+        iso_lang: String,
+        ps_version: String,
```

Comment:
Better to use camelCase for props names. See: https://vuejs.org/v2/style-guide/#Prop-name-casing-strongly-recommended

Reply:
Done!